solr/core/src/test/org/apache/solr/TestDistributedGrouping.java

```diff
@@ -221,12 +224,13 @@ public void test() throws Exception {
         "fl", "id", "group.format", "grouped", "group.limit", "-12",
         "sort", i1 + " asc, id asc");
 
+    ignoreException("'group.offset' parameter cannot be negative");
```

Comment:
If we are doing this, shouldn't we call `resetExceptionIgnores()` after exception verification?

Reply:
Good point, added `resetExceptionIgnores`. Thanks!